### PR TITLE
Change Spek JVM runtime to create path one time only on discovery.

### PR DESCRIPTION
**Pull request by arturdryomov**

### CODE REVIEW

**Review by raniejade — APPROVED** [review]

**Comment by artem-zinnatullin** [comment]
Uhh, TC just doesn't see this branch no matter what I do

**Review by artem-zinnatullin — APPROVED** [review]

File: spek-runtimes/jvm/src/main/kotlin/org/spekframework/spek2/runtime/SpekJvmRuntime.kt [hunk 1]
```diff
@@ -36,17 +36,14 @@ actual class SpekRuntime: AbstractRuntime() {
             .map(Class<out Spek>::kotlin)
             .filter { it.findAnnotation<Ignore>() == null }
             .filter { !it.isAbstract }
-            .filter {
-                val path = PathBuilder.from(it)
-                    .build()
-
+            .map { klass ->
+                klass to JvmPathBuilder.from(klass).build()
```
Comment (artem-zinnatullin) [inline spek-runtimes/jvm/src/main/kotlin/org/spekframework/spek2/runtime/SpekJvmRuntime.kt]:
```console
Unresolved reference: JvmPathBuilder
```

There is no more `JvmPathBuilder`, it's now just `PathBuilder` implemented for every platform :)

**Review by artem-zinnatullin — CHANGES_REQUESTED** [review]
Can you please update code to use `PathBuilder`?

**Comment by artem-zinnatullin** [comment]
CI to the rescue!

**Comment by arturdryomov** [comment]
@artem-zinnatullin, damn, these kids grow up fast. Should be fine by now.

**Review by artem-zinnatullin — APPROVED** [review]

**Pull request merged**